src/elements/Collapse/Collapse.tsx

```diff
@@ -0,0 +1,35 @@
+import React from "react"
+import { animated, Spring } from "react-spring"
+
+export interface CollapseProps {
+  /** is open? */
+  open: boolean
+}
+
+/** Collapse component */
```

Comment:
This comes through in VSCode tooling and eventually docs; mind expanding this decription a bit?